valkyrie/values.yaml

```diff
@@ -0,0 +1,41 @@
+replicaCount: 2
+
+image:
+  repository: smartcitiesdata/valkyrie
+  tag: development
+  pullPolicy: Always
+
+# monitoring:
+  # targetPort: 4000
+# If running locally on Minikube, set to true to deploy an AWS localstack instance
+# https://github.com/localstack/localstack
+localstack:
+  enabled: false
+
+kafka:
+  #comma delimited list of kafka seed brokers
+  brokers: streaming-service-kafka-bootstrap.streaming-prime:9092
```

Comment:
were we genericizing ala https://github.com/smartcitiesdata/charts/pull/47/files#diff-134cad1b24d44470f8f6de276dda2031R13 these or did we decide we didn't care?

Reply:
If that standardization is going to happen it will probably happen with the mega-chart.

Reply:
Fair enough 👎